resources/js/components/Listing/ListingView.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+    <div>
+        <div v-if="initializing" class="card loading">
+            <loading-graphic />
+        </div>
+
+        <data-list
+            v-if="!initializing"
+            class="mb-4"
+            :visible-columns="columns"
+            :columns="columns"
+            :rows="items"
+            :sort="false"
+            :sort-column="sortColumn"
+            :sort-direction="sortDirection"
+        >
+            <div slot-scope="{ hasSelections }">
+                <div class="card p-0 relative">
+                    <data-list-filter-presets
+                        ref="presets"
+                        :active-preset="activePreset"
+                        :preferences-prefix="preferencesPrefix"
+                        @selected="selectPreset"
+                        @reset="filtersReset"
+                    />
+
+                    <div class="data-list-header">
+                        <data-list-filters
+                            :filters="filters"
+                            :active-preset="activePreset"
+                            :active-preset-payload="activePresetPayload"
+                            :active-filters="activeFilters"
+                            :active-filter-badges="activeFilterBadges"
+                            :active-count="activeFilterCount"
+                            :search-query="searchQuery"
+                            :saves-presets="true"
+                            :preferences-prefix="preferencesPrefix"
+                            @filter-changed="filterChanged"
+                            @search-changed="searchChanged"
+                            @saved="$refs.presets.setPreset($event)"
+                            @deleted="$refs.presets.refreshPresets()"
+                            @restore-preset="$refs.presets.viewPreset($event)"
+                            @reset="filtersReset"
+                        />
+                    </div>
+
+                    <div
+                        v-show="items.length === 0"
+                        class="p-3 text-center text-grey-50"
+                        v-text="__('No results')"
+                    />
+
+                    <data-list-table
+                        v-show="items.length"
+                        :allow-bulk-actions="false"
+                        :loading="loading"
+                        :reorderable="false"
+                        :sortable="true"
+                        :toggle-selection-on-row-click="true"
+                        :allow-column-picker="true"
+                        :column-preferences-key="preferencesKey('columns')"
+                        @sorted="sorted"
+                    >
+                        <template slot="cell-label" slot-scope="{ row }">
+                            <a
+                                :href="cp_url(listingConfig.editUrl + row.id)"
```

Comment:
We seem to be taking the `editUrl` passed from the `listingConfig` and just adding to it here. Is there any way we can generate this for each row rather than doing it this way?